plugins/action/include/plugins/action/action.h

```diff
@@ -106,6 +106,20 @@ class Action : public PluginBase {
      */
     ActionResult return_to_launch() const;
 
+    /**
+     * @brief Send command to reposition the vehicle to a specific WGS84 global position
+     *
+     * This sends the vehicle to a specified lattitude/longitude/altitude coordinates.
+     * @param latitude_deg Latitude in degrees
```

Comment:
FYI, I wouldn't necessarily change this now, but one of the benefits of naming with units is that you don't necessarily have to spell everything out in the description. So normally I'd just have `Latitude.` (and yes, arguably you don't even need that, but good to have something because it shows that you have thought about docs).